android/src/main/java/com/thebluealliance/androidclient/helpers/ThreadSafeFormatters.java

```diff
@@ -55,4 +55,8 @@ public static synchronized String formatDoubleOnePlace(double input) {
     public static synchronized String formatDoubleTwoPlaces(double input) {
         return DOUBLE_TWO_PLACES_FORMAT.format(input);
     }
+
+    public static synchronized String formatStat(double stat) {
```

Comment:
Nit: this is exactly the same as `formatDoubleTwoPlaces`, why not just use that method?